mqtt_bridge/bridge.py

```diff
@@ -19,10 +21,10 @@ def create_bridge(factory: Union[str, "Bridge"], msg_type: Union[str, Type[rospy
         raise ValueError("factory should be Bridge subclass")
     if isinstance(msg_type, str):
         msg_type = lookup_object(msg_type)
-    if not issubclass(msg_type, rospy.Message):
+    """if not issubclass(msg_type, rospy.Message):
```

Comment:
this seems unnecessary, too. or can you replace the code with ROS2 syntax?

Reply:
Right now, a solution for this does not exist in ROS2. but I want to keep these comments so that people could update it in the future.

Reply:
I see. Please use
```
# xxxxx
# xxxxx
```
syntax for commenting out, since docstring is for documentation.

Reply:
This has been changed locally will push a new commit for this